src/UpdateLocalizedResources.targets

```diff
@@ -1,27 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="UpdateXlf" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   
-  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" Condition="$(LocalizedBuild) == 'true'" TaskName="SaveXlfToResx"/>
-  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" Condition="$(LocalizedBuild) == 'true'" TaskName="UpdateXlfFromResx"/>
-  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" Condition="$(LocalizedBuild) == 'true'" TaskName="EmitLocalizedResources"/>
+  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" TaskName="SaveXlfToResx"/>
+  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" TaskName="UpdateXlfFromResx"/>
+  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" TaskName="EmitLocalizedResources"/>
+
+  <Target Name="AddLocalizedResxFilesToEmbeddedResource"
+          BeforeTargets="$(PrepareResourcesDependsOn);PrepareResources"
+          DependsOnTargets="SyncXlf;GenerateLocalizedResxResourcesFromXlf;AddEnglishResxFilesToEmbeddedResource"
+          Condition="'$(LocalizedBuild)' == 'true'">
 
-  <!-- Emit the localized EmbeddedResource items before the common targets start using them -->
-  <Target Name="EmitLocalizedResources"
-          BeforeTargets="$(PrepareResourceNamesDependsOn);PrepareResourceNames">
     <ItemGroup>
-      <NeutralResources Include="@(EmbeddedResource)" Condition="'%(Extension)' == '.resx'"/> 
+      <EmbeddedResource Include="@(LocalizedResxResources)"/>
```

Comment:
LocalizedResxResources represents all the generated, non English localized resx files from the xlf files. The dependency of this target AddEnglishResxFilesToEmbeddedResource does the English workaround and adds itself to EmbeddedResource